fuse_examples/imaging/segmentation/picai/runner.py

```diff
@@ -0,0 +1,381 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Created on June 30, 2021
+"""
+
+import os
+import copy
+from pathlib import Path
+import pickle
+import pandas as pd
+import numpy as np
+from fuse.utils.utils_debug import FuseDebug
+from fuse.utils.gpu import choose_and_enable_multiple_gpus
+
+import logging
+
+import torch.optim as optim
+from torch.utils.data.dataloader import DataLoader
+
+from fuse.utils.utils_logger import fuse_logger_start
+from fuse.utils import NDict
+from fuse.data.utils.samplers import BatchSamplerDefault
+from fuse.data.utils.collates import CollateDefault
+from fuse.data.utils.split import dataset_balanced_division_to_folds
+from fuse.dl.losses.loss_default import LossDefault
+
+# from report_guided_annotation import extract_lesion_candidates
+import monai
+from fuse_examples.imaging.segmentation.picai.unet import UNet
+
+from fuse.eval.metrics.segmentation.metrics_segmentation_common import MetricDice
+from collections import OrderedDict
+
+# from fuse.eval.metrics.detection.metrics_detection_common import MetricDetectionPICAI
+from fuseimg.datasets.picai import PICAI
+from fuse.dl.lightning.pl_funcs import convert_predictions_to_dataframe
+from pytorch_lightning import Trainer
+from fuse.dl.lightning.pl_module import LightningModuleDefault
+from fuse.utils.file_io.file_io import create_dir, load_pickle, save_dataframe
+from fuse.eval.evaluator import EvaluatorDefault
+
+# from picai_baseline.unet.training_setup.default_hyperparam import \
+#     get_default_hyperparams
+# from picai_baseline.unet.training_setup.neural_network_selector import \
+#     neural_network_for_run
+import torch
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+# from monai.losses import DiceFocalLoss
+
+# assert (
+#     "PICAI_DATA_PATH" in os.environ
+# ), "Expecting environment variable PICAI_DATA_PATH to be set. Follow the instruction in example README file to download and set the path to the data"
+##########################################
+# Debug modes
+##########################################
+mode = "default"  # Options: 'default', 'debug'. See details in FuseDebug
+debug = FuseDebug(mode)
+
+
+def create_model(unet_kwargs: dict) -> torch.nn.Module:
+    """
+    creates the model
+    """
+    model = UNet(
+        input_name="data.input.img_t2w",
+        seg_name="model.seg",
+        unet_kwargs=unet_kwargs,
+    )
+
+    return model
+
+
+#################################
+# Train Template
+#################################
+def run_train(paths: NDict, train: NDict) -> None:
+    # ==============================================================================
+    # Logger
+    # ==============================================================================
+    # fuse_logger_start(output_path=paths["model_dir"], console_verbose_level=logging.INFO)
```

Comment:
You can remove the fuse_logger_start and move to standard prints. 
We will deprecate it soon.

Reply:
ok